src/decision/dMCDA.jl

```diff
@@ -3,9 +3,11 @@ module decision
 
 using StatsBase
 using Distances, DataFrames
+using DimensionalData
```

Comment:
Is this import necessary?

If it's just for the `At` method, I prefer it's imported directly:

`import YAXArrays.DD: At`

Reply:
Apparently it doesn't make any difference (I've removed it and everything worked fine).